sfc/application.txt

```diff
@@ -0,0 +1,632 @@
+    collaborate occasionally on business matters like discussing
+    trademarks and the like.
+
+    This coordination has informed our desire to join the Conservancy,
+    and we feel it a natural outgrowth of our coordination with Git.
+
+*** Please give a detailed description of the project.
+
+    The libgit2 project is a portable, pure C implementation of the Git
+    core methods provided as a re-entrant linkable library.  It is used
+    for the Git repository management functionality in a number of
+    different services and products, both open source and proprietary.
+
+    Most, if not all, of these products use libgit2 by way of one of a
+    number of "language binding" libraries that are built on top of libgit2
+    and exist to provide idiomatic Git libraries in other languages.
+
+    While each of these language binding projects have their own
+    development communities around them, they each have significant
+    interaction with the libgit2 community.  Many of the developers and
+    community members of the libgit2 project also work on the language
+    bindings.  The most popular and developed of these are:
+
+    - LibGit2Sharp, a .NET implementation on top of libgit2
+    - NodeGit, a node.js implementation on top of libgit2
+    - ObjectiveGit, an Objective C implementation on top of libgit2, and
+    - Rugged, a Ruby implementation on top of libgit2
+
+*** What FLOSS License(s) does your project use?  Please include the
+    primary license, and list other licenses for code that is included.
+    (e.g., "The project as a whole is GPLv3-or-later, but about a dozen
+    files in the directory src/external/ are under the Apache-2.0
+    license").  Please be sure to include information on documentation
+    licensing as well as software licensing.
+
+    libgit2 is licensed under the GPL version 2 with a linking
+    exception.  The linking exception is:
+
+        In addition to the permissions in the GNU General Public License,
+        the authors give you unlimited permission to link the compiled
+        version of this library into combinations with other programs,
+        and to distribute those combinations without any restriction
+        coming from the use of this file.  (The General Public License
+        restrictions do apply in other respects; for example, they cover
+        modification of the file, and distribution when not linked into
+        a combined executable.)
+
+    In addition, we include some additional small components that are
+    under different licenses:
+
+    src/khash.h: MIT
+    src/path.c:git_path_basename_r: BSD
+    src/path.c:git_path_dirname_r: BSD
+    src/tsort.c: MIT
+    src/util.c:git__bsearch: BSD
+    src/util.c:git__utf8_iterate: BSD
+
+    Finally, we include some dependencies for cross-platform compilation
+    that are included in the project when they are not available on the
+    underlying operating system:
+
+    src/fnmatch.c: BSD
```

Comment:
This is outdated, we've migrated to "wildmatch.c" from the Git project. It's written by Rich Salz originally with no explicit license, but IIRC we've explicitly asked him and got the permission to relicense under GPLv2.